.storybook/docs-root.css

```diff
@@ -1,24 +1,26 @@
-#docs-root .sbdocs-content {
+#storybook-docs .sbdocs-content {
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32018&commitId=c16b9699b21bd898b43cc776da7b018bc4315cb8&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Image.normal.chromium.png|248| Changed|

</details>


  <details><summary><b>Checkbox</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Circular Indeterminate - Dark Mode.indeterminate.chromium.png|0| Removed|
|Checkbox.Circular Indeterminate.indeterminate.chromium.png|0| Removed|
|Checkbox.Indeterminate - Dark Mode.indeterminate.chromium.png|0| Removed|
|Checkbox.Indeterminate.indeterminate.chromium.png|0| Removed|

</details>


  <details><summary><b>Tabs</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tabs.Navigation Vertical.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>